linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/ServerConfiguration.scala

```diff
@@ -65,19 +65,19 @@ object ServerConfiguration extends Logging{
   val BDP_TEST_USER = CommonVars("wds.linkis.test.user", "")
 
   val BDP_SERVER_HOME = CommonVars("wds.linkis.server.home", CommonVars("LINKIS_HOME", "").getValue)
-  val BDP_SERVER_DISTINCT_MODE = CommonVars("wds.linkis.server.distinct.mode", new Boolean(true))
+  val BDP_SERVER_DISTINCT_MODE: CommonVars[lang.Boolean] = CommonVars("wds.linkis.server.distinct.mode", lang.Boolean.TRUE)
```

Comment:
Why is there no unified style for Boolean here?

Reply:
this Boolean is used by java,   scala boolean can not be recognition